lib/system/storage.toit

```diff
@@ -419,6 +419,9 @@ class RegionReader_ implements Reader:
     from_ = from + n
     return result
 
+  content-size -> int?:
+    return null
```

Comment:
Maybe add a TODO here? It would be kinda nice to be able to get the content-size for this and we do know it.

Reply:
done.